soccer/radio/SimRadio.cpp

```diff
@@ -41,18 +45,24 @@ void SimRadio::send(Packet::RadioTx& packet) {
         // corresponds to 8 m / s and min is 1 m / s
         const float min_kick_m_s = 2.1f;
         const float max_kick_m_s = 7.0f;
+        const float min_chip_m_s = 2.1f;
+        const float max_chip_m_s = 5.0f;
+        const float chip_angle = 40*PI/180; // degrees
```

Comment:
We should have a deg2rad somewhere in common/util or something

Reply:
I think we do, somewhere - I remember it being used in the geometry matrix in fshare at least. I'll take a look.